install-linux.sh

```diff
@@ -282,7 +282,13 @@ update_config_files() {
 		sed -i "/^CYRAL_PUSH_CLIENT_FQDN=/c\CYRAL_PUSH_CLIENT_FQDN=\"${CYRAL_SIDECAR_ID}\"" /etc/default/cyral-push-client
 		# fix legacy ports
 		sed -i "s/8050/8069/" /etc/default/cyral-push-client
-
+	else
+	cat > /etc/default/cyral-push-client <<EOF
+CYRAL_PUSH_CLIENT_FQDN="${CYRAL_SIDECAR_ID}"
+CYRAL_PUSH_CLIENT_PROXY_URL=http://localhost:8069
+ENDPOINTS=['localhost:8068']
+TIMEOUT=5
```

Comment:
It might be safer/cleaner that instead of creating the file in the else block, we could do it before the if block above with the default contents if the file does not already exist, and then let the existing code do what it's doing now. What do you think?

For example, the code in the else block is setting TIMEOUT which is not happening in the if block. Also the first sed command is not taking effect in the else case either.

Reply:
It is cleaner, but I'm not sure if it is safer as we have several different versions of sidecar still running.